compat/vcbuild/vcpkg_install.bat

```diff
@@ -34,6 +34,12 @@ REM ================================================================
 	@FOR /F "delims=" %%D IN ("%~dp0") DO @SET cwd=%%~fD
 	cd %cwd%
 
+	SET arch=x64-windows
```

Comment:
Is this really the correct thing to do? I'd rather have this as independent of the current architecture as possible.

Reply:
> Is this really the correct thing to do? I'd rather have this as independent of the current architecture as possible.

It's the current way. That appears to be just the way it is (see lower down in the bat file).

Reply:
As I mentioned below, if we are already moving it, we should move it to the top, and add a good comment about the raison d'être for this variable.